index.html

```diff
@@ -182,6 +182,25 @@ <h2> About <br> Myself</h2>
             </ul>
           </div>
           <hr>
+
+          <section class="section-form">
+            <div class="form-div">
+             <h3>Contact me</h3>
+             <p>If you have an application you are 
+               interested in developing,
+               a feature that you need built or a project that 
+               needs coding. I’d love to help with it </p>
+            </div>
+            <form action="/my-handling-form-page" method="post">
```

Comment:
- [ ] While you did a wonderful job using the `method` attribute :+1:, as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) the `action` attribute should be similar to `https://formspree.io/f/{form_id}` to integrate and collect data with [formspree service](https://formspree.io/html)